src/python/pants/backend/codegen/thrift/python/py_thrift_namespace_clash_check.py

```diff
@@ -26,9 +26,13 @@ class PyThriftNamespaceClashCheck(Task):
   @classmethod
   def register_options(cls, register):
     super().register_options(register)
-    # TODO: deprecate the --strict option in a future release!
+    # TODO: deprecate the --strict option in a future release, and strict should be always true.
```

Comment:
Should probably deprecate this now, and recommend the other options?

Reply:
Thanks! Adjusted.